jsonp/1.1/_index.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "Jakarta JSON Processing Specification 1.1"
+date: 2019-07-19T11:00:00+0100
+summary: "First release for Jakarta EE 8"
+---
+Jakarta JSON Processing defines a Java(R) based framework for parsing, generating, transforming, and
+querying JSON documents.
+
+* [Jakarta JSON Processing 1.1 Specification Document](./jsonp_1.1.pdf) (PDF)
+* [Jakarta JSON Processing 1.1 Specification Document](./jsonp_1.1.html) (HTML)
+* [Jakarta JSON Processing 1.1 Javadoc](./apidocs)
+* [Jakarta JSON Processing 1.1 TCK](http://download.eclipse.org/ee4j/jsonp/jsonp-tck/jsonp-tck-eftl/release/eclipse-jsonptck-1.0.0.zip)
```

Comment:
This should be the URL of the location where the signed version will be published:
download.eclipse.org/jakartaee/jsonp/1.1/eclipse-jsonp-tck-1.1.0.zip

Reply:
Fixed